filestore/protoboards.go

```diff
@@ -0,0 +1,118 @@
+package filestore
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/influxdata/chronograf"
+)
+
+// ProtoboardExt is the the file extension searched for in the directory for protoboard files
+const ProtoboardExt = ".json"
+
+// Protoboards are instantiable JSON representation of dashbards.  Implements ProtoboardsStore.
+type Protoboards struct {
```

Comment:
Why not call this `ProtoboardStore`?

Reply:
copied from layouts. happy to modernize.